theano/tensor/elemwise.py

```diff
@@ -477,14 +476,16 @@ class Elemwise(OpenMPOp):
 
     """
 
+    __props__ = ("scalar_op", "inplace_pattern", "name", "nfunc_spec", "openmp")
```

Comment:
When I'm removing name from the `__init__` and `__props__`, the string will change and I have to accordingly change the ground truths in the node assertion tests. Is that fine @nouiz ?